service/src/main/resources/api/service_openapi.yaml

```diff
@@ -145,6 +145,80 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/profiles/v1/{profileId}/policies:
+    parameters:
+      - $ref: '#/components/parameters/ProfileId'
+    get:
+      tags:
+        - Profile
+      summary: Retrieve policies for the profile
+      operationId: getProfilePolicies
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SamPolicyModelList'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+
+  /api/profiles/v1/{profileId}/policies/{policyName}/members/{memberEmail}:
+    post:
+      tags:
+        - Profile
+      description: >
+        Adds a member to the specified policy for the profile.
+        NOTE: Change may take up to 60 seconds to take effect.
+      operationId: addProfilePolicyMember
+      parameters:
+        - $ref: '#/components/parameters/ProfileId'
+        - $ref: '#/components/parameters/ProfilePolicyName'
+        - $ref: '#/components/parameters/MemberEmail'
+      responses:
+        201:
+          description: Policy
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SamPolicyModel'
```

Comment:
The jade-data-repo versions return a list of policies, but given that there will only ever be one, that seems unnecessarily complex.